DOCUMENTATION.md

```diff
@@ -135,12 +204,32 @@ Enable device url for a device.
 
 #### Raises:
     DeviceNotFound: if device couldn't be found.
+
+#### Examples:
+    Check if device url enabled.
+```python
+>>> resin.models.device.has_device_url('8deb12a58e3b6d3920db1c2b6303d1ff32f23d5ab99781ce1dde6876e8d143')
+... False
+Enable device url.
+>>> resin.models.device.enable_device_url('8deb12a58e3b6d3920db1c2b6303d1ff32f23d5ab99781ce1dde6876e8d143')
+... 'OK'
+Check device url again.
+>>> resin.models.device.has_device_url('8deb12a58e3b6d3920db1c2b6303d1ff32f23d5ab99781ce1dde6876e8d143')
+... True
+```
```

Comment:
Reformat the whole block as:
```python
# Check if device url enabled.
>>> resin.models.device.has_device_url('8deb12a58e3b6d3920db1c2b6303d1ff32f23d5ab99781ce1dde6876e8d143')
False
# Enable device url.
>>> resin.models.device.enable_device_url('8deb12a58e3b6d3920db1c2b6303d1ff32f23d5ab99781ce1dde6876e8d143')
'OK'
# Check device url again.
>>> resin.models.device.has_device_url('8deb12a58e3b6d3920db1c2b6303d1ff32f23d5ab99781ce1dde6876e8d143')
True
```

(Don't forget to put "```python" around it -- I omitted them because I couldn't escape the tick marks and it would break the comment formatting)